src/data_availability/node_state/verifiers.rs

```diff
@@ -35,15 +58,10 @@ pub fn risc0_proof_verifier(encoded_receipt: &[u8], image_id: &[u8]) -> Result<H
 
     let hyle_output = receipt
         .journal
-        .decode::<HyleOutput>()
+        .decode::<T>()
         .context("Failed to extract HyleOuput from Risc0's journal")?;
 
-    tracing::info!(
-        "✅ Risc0 proof verified. {}",
-        std::str::from_utf8(&hyle_output.program_outputs)
-            .map(|o| format!("Program outputs: {o}"))
-            .unwrap_or("Invalid UTF-8".to_string())
-    );
```

Comment:
Was usefull when debugging :/

Reply:
I'll add it back at the `verify_proof` level because this is actually proof-system agnostic